purchase_reception_status/models/purchase_order.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    reception_status = fields.Selection([
+        ('no', 'Nothing Received'),
```

Comment:
add a new option in case the purchase order only has services:
 ('no_rec', 'Nothing to Received'),

Reply:
I disagree, for the same reason as above.